pegasus/sites.v3/code.org/public/learn/index.haml

```diff
@@ -131,7 +131,7 @@ style_min: false
       locale: "#{request.locale}",
       roboticsButtonUrl: "/learn/robotics",
       showSortDropdown: #{DCDO.get('learn_show_sort_dropdown', true)},
```

Comment:
Could the same issue appear on other lines, such as here?

Reply:
it could in theory, but my expectation is that this field is supposed to contain only a boolean value, which won't get escaped and so doesn't need to be explicitly unescaped.

I don't have a strong opinion about whether we should unescape proactively or cautiously.